pkg/build/api/types.go

```diff
@@ -177,6 +190,9 @@ const (
 	// StatusReasonExceededRetryTimeout is an error condition when the build has
 	// not completed and retrying the build times out.
 	StatusReasonExceededRetryTimeout = "ExceededRetryTimeout"
+
+	// StatusReasonSkipped indicates that the build was skipped.
+	StatusReasonSkipped = "Skipped"
```

Comment:
for now the only reason a build will be skipped is that the user took a manual action to skip over all the queued builds in order to run the most recent build request.